### ISSUE TITLE
How do you train with those NOT-TEXT elements.

### ISSUE THREAD

**Issue by linan142857** [doc-analysis/DocBank#21]
Dear author,
    For some documents that contain massive not-text elements, such as hundreds of thousands of "##LTLine##". How do you deal with them actually? 
    For example, you try to train&predict all those elements with text '##LTLine##'.

Thank you!
![image](https://user-images.githubusercontent.com/3764907/98514672-cc26ae00-22a4-11eb-8695-12cebe677a98.png)

**Comment by liminghao1630** [comment]
Yes.  We regard '##LTLine##' as a special token during train and predict.

**Comment by NandreyN** [comment]
> Yes. We regard '##LTLine##' as a special token during train and predict.

Hi! Could you please tell integer identifiers of `##LTLine##` and `##LTFigure##`  tokens within LayoutLM's vocabulary?

Thanks

**Comment by liminghao1630** [comment]
In fact, we did not add them to the vocabulary. They will also be tokenized into tokens and labeled in the way I mentioned at #25.

**Comment by NandreyN** [comment]
Thanks
